tests/common.bash

```diff
@@ -159,12 +163,28 @@ function extract_kata_env() {
 			hypervisor_path=".Hypervisor.Path"
 			virtio_fs_daemon_path=".Hypervisor.VirtioFSDaemon"
 			initrd_path=".Initrd.Path"
+			shared_fs=".Hypervisor.SharedFS"
```

Comment:
Hi @dborquez - was there are reason that `shared_fs` wasn't set on dragonball as this is also causing failures with `shared_fs: unbound variable` in https://github.com/kata-containers/kata-containers/actions/runs/9900211956/job/27350849536